### DISCUSSION TITLE
Another Bypass

### DISCUSSION THREAD

**Discussion by Aka-but-nice** [3kh0/ext-remover#348] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Go to this link https://stream-mysterious-bathroom.glitch.me/
bookmark the code there then go to tinyurl.com/goofcurly if you have securly, tinyurl.com/goofguardian if you have goguardian, tinyurl.com/goofsi if you have blocksi tinyurl.com/goofclassroom if you have securly classroom, each of the links have a button on the pages and each open a about:blank tab click on the bookmarklet you created. If you want a video, for documentary, just state it. Also, if you go to the link of the blocker like tinyurl.com/goofcurly it might say this link is blocked by chrome, just reload it, and credit the Titanium Network Discord

**Comment by obkduishinijf** [comment]
problem, check #366

**Reply by Aka-but-nice** [reply]
its because the link is not a web-accessible resource, so for it to work use it in a new tab and if it says blocked

**Reply by Aka-but-nice** [reply]
by chrome just reload

**Comment by troycla** [comment]


**Reply by troycla** [reply]
i am going to help with the unblocker js some people can not use javascript for bookmarklet the school can block it
proxys are best opstion right now

**Reply by troycla** [reply]
i viewing your code and run it there is muiltple problems
first does not ho a sortin domin to inject and second of all
the code does not work at all

**Reply by obkduishinijf** [reply]
what? did you have a stroke trying to type that

**Reply by Aka-but-nice** [reply]
> i viewing your code and run it there is muiltple problems first does not ho a sortin domin to inject and second of all the code does not work at all

@troycla But did I ask. You are 5 years old, you have no knowledge of what your talking about, you are speaking shadow realm. You have a base IQ of 16, you can't spell for the life of you, you can't even spell multiple, my IQ is 20 fold of yours. Stop playing yourself.

**Reply by Aka-but-nice** [reply]
Do you even know what XSS is, naw don't try to look it up now, tell me.

**Reply by troycla** [reply]
bro am 16 teen Here is a fix to your problem
[https://github.com/troycla/kill-exetsions-](url)

**Reply by troycla** [reply]
PS don't blame me for your problems

**Reply by ehunt34** [reply]
Bro really said he has 320 IQ when the record is 228.

**Reply by Aka-but-nice** [reply]
Skill issue

**Comment by ItsLumine** [comment]
doesn't work for me

**Reply by Aka-but-nice** [reply]
For what extension?

**Comment by PSYclownYT** [comment]
what if you use lightspeed filter agent

**Reply by PSYclownYT** [reply]
Thanks! But I figured it out already lol. It's not the bookmark it's the running on about-blank launched by the privileged page.
